docs/src/main/asciidoc/kogito-pmml.adoc

```diff
@@ -0,0 +1,255 @@
+
+This guide demonstrates how your Quarkus application can use Kogito to add business automation
+to power it up with predictions.
+
+Kogito is a next generation business automation toolkit that originates from well known Open Source projects
+Drools (for predictions). Kogito aims at providing another approach
+to business automation where the main message is to expose your business knowledge (processes, rules, decisions, predictions)
+in a domain specific way.
+
+include::./status-include.adoc[]
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 11+ installed with `JAVA_HOME` configured appropriately
+* Apache Maven {maven-version}
+* Docker
+
+== Architecture
+
+In this example, we build a very simple microservice which offers one REST endpoint:
+
+* `/LogisticRegressionIrisData`
+
+This endpoint will be automatically generated based on given PMML file, that in turn will
+make use of generated code to make certain predictions based on the data being processed.
+
+=== PMML file
+
+The PMML file describes the prediction logic of our microservice.
+It should provide the actual model (Regression, Tree, Scorecard, Clustering, etc) needed to make the prediction.
+
+=== Prediction endpoints
+
+Those are the entry points to the service that can be consumed by clients.
+
+== Solution
+
+We recommend that you follow the instructions in the next sections and create the application step by step.
+However, you can go right to the complete example.
+
+Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+
+The solution is located in the `kogito-pmml-quickstart` {quickstarts-tree-url}/kogito-pmml-quickstart[directory].
+
+== Creating the Maven Project
+
+First, we need a new project. Create a new project with the following command:
+
+[source,bash,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=kogito-pmml-quickstart \
+    -Dextensions="kogito" \
+    -DnoExamples
+cd kogito-pmml-quickstart
+----
+
+This command generates a Maven project, importing the `kogito` extension
+that comes with all needed dependencies and configuration to equip your application
+with business automation.
+
+If you already have your Quarkus project configured, you can add the `kogito` extension
+to your project by running the following command in your project base directory:
+
+[source,bash]
+----
+./mvnw quarkus:add-extension -Dextensions="kogito"
+----
+
+This will add the following to your `pom.xml`:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-quarkus</artifactId>
+</dependency>
+----
+
+== Writing the application
+
+Let's start by adding a simple PMML file `LogisticRegressionIrisData.pmml`. As you can see from the source code below it contains a _Regression_ model named `LogisticRegressionIrisData`:
+
```

Comment:
I would add a small description of what this file is supposed to do. Might not be obvious for people not accustomed to Kogito (but it might not be your audience so I'll let you judge of that).

Reply:
Hi @gsmet 
"pmml" is a standard unrelated to Kogito (like, dmn, drl, doc, xlm, etc), and "Predictions" is the name we are using inside Kogito for it. TBH I'm expecting people going to try "Prediction" to be perfectly aware of it, but you may have a point...

Reply:
Yeah, it's more if you expect to have curious people around :).

Reply:
@gsmet 
Just done! Please let me know what you think: if it is fine for you, it could be merged.

Reply:
Spotted a small typo, otherwise it looks good.